app/javascript/components/patient/Patient.js

```diff
@@ -291,7 +290,7 @@ class Patient extends React.Component {
               id="details-expander-link"
               variant="link"
               className="p-0"
-              aria-label="Show address, travel, exposure, and case information"
+              aria-label={`${this.state.expanded ? 'Collapse' : 'Expand'} address, travel, exposure, and case information`}
```

Comment:
Makes it clearer to the user which state the UI is in

Reply:
good catch